docs/src/main/sphinx/object-storage/file-system-cache.md

```diff
@@ -106,6 +106,11 @@ enable and configure caching for the specific catalogs.
     caching. All directories must exist on the coordinator and all workers.
     Trino must have read and write permissions for files and nested directories.
     A valid example with only one directory is `/tmp/trino-cache`.
+
+    Directories must be specific for each catalog with caching enabled. When
```

Comment:
I know the two sentences kind of say the same thing twice .. the idea is that it drives the point more .. but maybe its overkill and we can leave the first sentence out

Reply:
I like it! It's very clear 👍